### ISSUE TITLE
Merge tabs in basic mode

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#787]
If a user is running UMS in basic mode (meaning they are hiding the advanced settings) I think we should merge the Transcoding Settings tab (which only has one option) and the Navigation/Share settings tab (which only has folder selection) into the General Configuration tab which can be renamed to Configuration.

It's a relatively easy/fast way to make the interface easier to use for users in that mode.

**Comment by Nadahar** [comment]
I agree, the GUI doesn't make much sense in "basic" mode. But, I'd like for us to fully implement ```ViewLevel``` in the same instance. I made that in #616, but it's currently only used on the ```Logs``` tab.

The difference is that we have multiple "view levels" ("normal/basic", advanced, expert, developer) so that a user can use "advanced" without being overwhelmed with the most cryptic settings in the GUI. At the same time I threw in "developer" if we want to throw in some really obscure settings for testing things.

I also made it easy to define what view level each component should have, and that they are hidden and shown without restarting UMS. This is all used on the ```Logs``` tab, and I'd just like to expand it to the other tabs and replace the "hide advanced settings" checkbox with a "view level" combobox.
